runtime/oti/VMHelpers.hpp

```diff
@@ -2061,6 +2066,18 @@ class VM_VMHelpers
 #endif /* JAVA_SPEC_VERSION >= 19 */
 		return needScan;
 	}
+
+	static VMINLINE void
+	cleanupContinuationObject(J9VMThread *vmThread, J9Object *objectPtr)
+	{
+#if JAVA_SPEC_VERSION >= 19
+		J9VMContinuation *j9vmContinuation = J9VMJDKINTERNALVMCONTINUATION_VMREF(vmThread, objectPtr);
+		if (NULL != j9vmContinuation) {
+			/* clean up J9VMContinuation, set vmref = NULL */
+			J9VMJDKINTERNALVMCONTINUATION_SET_VMREF(vmThread, objectPtr, NULL);
```

Comment:
I suspect we have to do something more here, some kind of free operation against j9vmContinuation

Reply:
yes, vm will add further clean up here.

Reply:
@tajila please check a couple of VM files in this PR